storybookjs/aem
# Refactoring Render method

Issue:
Refactoring Render method.

## What I did
- Removed `dedent` from render file in favor of simpler `array.join()`
- Refactored render method to - hopefully - be easier to understand :)
- Added new text story to test the string template

## How to test
View the three different Text stories and inspect the output in browser
- The Text story has a custom wrapper element due to parameters on the story
- The Rich Text story has no wrapper element due to `decorationTag: null` parameters
- The String Test story has the default wrapper element

- Is this testable with Jest or Chromatic screenshots? Maybe?
- Does this need a new example in the kitchen sink apps? Has been updated
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
